corehq/apps/cloudcare/static/cloudcare/js/formplayer/js/apps/menus/menu_controller.js

```diff
@@ -52,12 +54,28 @@ FormplayerFrontend.module("SessionNavigate.MenuList", function (MenuList, Formpl
                 collection: detailCollection,
             });
 
+            var tabModel = [];
+            for (var j = 0; j < detailObjects.length; j++) {
+                var tabObject = {};
+                tabObject.title = detailObjects[j].title;
+                tabObject.id = j;
+                tabModel.push(tabObject);
+            }
+            var tabCollection = new Backbone.Collection();
+            tabCollection.reset(tabModel);
+            var tabListView = new MenuList.DetailTabListView({
+                collection: tabCollection,
+                showDetail: function(index) {
+                    myself(model, index);
```

Comment:
this is pretty tricky, can we make this `self.showDetail(model, index)` and set `var self = this` at the top of the file?